anet-dictionary.yml

```diff
@@ -472,9 +472,6 @@ dictionary:
               helpText: Help text for object date field
               visibleWhen: $[?(@.colourOptions === 'GREEN')]
 
-    location:
-      format: LAT_LON
```

Comment:
We still want to specify the default format in the dictionary. I suggest to leave it here and use it as the default.

Reply:
Hmm, if we are going to set a default there, we should also set all the possible formats as array of strings. Let me commit a poc for that and we can discuss on the implementation

Reply:
Now that i think about it more, defining array of formats in anet-dictionary.yml is probably useless. So one default format from the dictionary.

Reply:
Another question: When a user refreshes the webpage, even if he/she selected a format, the defaulFormat will be used. Should we maybe use local storage for keeping the format setting same between refreshes?

Reply:
> When a user refreshes the webpage
it should come back to the default.
We do not have user-specific settings and we do not want to change anything globally.

Actually the original request was to have the selection of coordinate type be a part of the coordinate widget itself. I think it is cleaner and more clear. Such as:

coordinate: 18SUJ7082315291 [MGRS] 

The last part being a dropdown on how to display the coordinates

Reply:
I think it is much simpler if it is self-contained in the coordinate widget

Reply:
Yes, i will include it to the location widget so we don't have to call it from multiple pages